src/styles/blocks/calendar.scss

```
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  box-sizing: border-box;
+
+  width: $calendarWidth;
+  padding: $calendarPadding;
+
+  @each $weekDay, $number in $weekDays {
+    &--start-day--#{$weekDay} &__day:first-child {
+      margin-left: $number * ($calendarWidth + $gap) / 7;
```

this seems more straightforward?
```suggestion
      margin-left: $number * ($sidesSize + $gap);
```